### PR TITLE
Added more content in map doc

### PR DESCRIPTION
**Pull request by ketankartoza**
This PR is for the issue #1703

@Jeremy-Prior  Please do not review this PR now, this PR relies on other PR I am creating this PR as a draft for now.

### CODE REVIEW

**Comment by ketankartoza** [comment]
@LunaAsefaw, I am not describing the year filter in this documentation because all the filters with the map are defined in the filter doc.

**Comment by ketankartoza** [comment]
@Jeremy-Prior  this PR is ready for review now.

File: docs/src/user/manual/explore/map.md [hunk 1]
```diff
@@ -21,11 +21,26 @@ This documentation outlines the functionalities available in the map view, inclu
 
 1. **Map**: The `Map` tab.
 
-2. **Zooming In Icon**: The zooming-in icon is represented by a plus sign (+). Click on this icon to zoom in on the map, making features and details appear larger and more visible.
+2. **Question Mark Icon**: Click on the `Question Mark Icon` to know the valuable insights into optimising the use of the Map page. Explore the helpful tips and guidelines provided in this sidebar to enhance the efficiency in navigating and utilising the map features. Click on the `visit our Documentation` link to visit the documentation
+    ![Side Bar](./img/map-4.png)
 
-3. **Zooming Out Icon**: The zooming out icon is represented by a minus sign (-). Click on this icon to zoom out on the map, reducing the size of the displayed area and providing a broader view.
+3. **Dots**: Upon landing, users are greeted with a visual representation of property locations in the form of dots, providing an immediate overview of the geographical distribution of properties (unless the user is designated as a data consumer).
```
Comment (Jeremy-Prior) [inline docs/src/user/manual/explore/map.md:27]:
```suggestion
3. **Points**: Upon landing on the page, users are greeted with a visual representation of property locations in the form of points, providing an immediate overview of the geographical distribution of properties (unless the user is designated as a data consumer).
```

**Review by Jeremy-Prior — CHANGES_REQUESTED** [review]
1 minor change

**Review by Jeremy-Prior — APPROVED** [review]
LGTM

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 8 min 20 sec

**Comment by lgkgh** [comment]
#### Jenkins Build Details:
**Build status:** SUCCESS
**Build duration:** 2 min 33 sec

**Pull request merged**